skbio/diversity/beta/_fast_unifrac.py

```diff
@@ -0,0 +1,444 @@
+from __future__ import division
+import numpy as np
+from ._unifrac import _validate
```

Comment:
I think all of this code should end up in ``._unifrac`` ultimately. We should refer to the Fast UniFrac paper in the docstrings, but I don't think we should name functions/files/etc with ``fast_unifrac``, but instead maybe just name the educational versions something like ``unweighted_unifrac_slow``.